src/commands/ext-configure.ts

```diff
@@ -48,21 +48,27 @@ export default new Command("ext:configure <extensionInstanceId>")
       const paramSpecWithNewDefaults = paramHelper.getParamsWithCurrentValuesAsDefaults(
         existingInstance
       );
-      const removedLocations = _.remove(paramSpecWithNewDefaults, (param) => {
-        return param.param === "LOCATION";
+      const immutableParams = _.remove(paramSpecWithNewDefaults, (param) => {
+        return param.immutable || param.param === "LOCATION";
+        // TODO: Stop special casing "LOCATION" once all official extensions make it immutable
       });
-      const currentLocation = _.get(existingInstance, "config.params.LOCATION");
+
       const params = await paramHelper.getParams(
         projectId,
         paramSpecWithNewDefaults,
         options.params
       );
-      if (removedLocations.length) {
+      let immutableParamsText = "";
+      for (const immutableParam of immutableParams) {
+        const currentValue = _.get(existingInstance, "config.params." + immutableParam.param);
+        immutableParamsText += `param: ${immutableParam.param}, value: ${currentValue}\n`;
+        params[immutableParam.param] = currentValue;
+      };
+      if (immutableParamsText) {
         logger.info(
-          `Location is currently set to ${currentLocation}. This cannot be modified. ` +
-            `Please uninstall and reinstall this extension to change location.`
```

Comment:
I'd change this line to:

"To set a different location, uninstall this extension, then install a new instance of this extension."